app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -29,14 +29,7 @@ def create
         Order::PlacedMailer.notification(order).deliver_later
         order.events.create(description: "Notification to Buyer Sent")
 
-        Stripe::Transfer.create({
-          # Leave the Stripe Fees in the `Distributor`'s account
-          amount: order.product_total.cents - balance_transaction.fee,
-          currency: "usd",
-          destination: marketplace.vendor_stripe_account,
-          transfer_group: order.id
-        }, {api_key: marketplace.stripe_api_key})
-        order.events.create(description: "Payment Split")
+        SplitJob.perform_later(order: order)
```

Comment:
Your point about Stripe holding the float for 3 days is an important one. What about setting this job to perform in 3 or 4 days? That way if the order gets cancelled or reversed or whatever, we don't do the transfer later.

```suggestion
        SplitJob.set(wait: 4.days).perform_later(order: order)
```

Reply:
I'm torn. On the one hand, waiting 3~4 days is not a bad idea since we don't want to transfer money out if we don't have it... On the other, forcing them to wait 7~10 days to receive payment for their goods feels not-great; since it will take 3~4 days from transfer to enter their accounts...

For now; I'd prefer to just try immediately; but I could be convinced otherwise.